src/stylesheets/App.css

```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
```

Comment:
- [ ] In the desktop home page, the items overlap each other and it's not possible to differentiate among each of them (this is showing 2 items):

![image](https://github.com/geekelo/fenaj-schedula/assets/57421823/220feff9-45fe-456a-a9fc-349c7c168894)

Same happens in the mobile view.


Kindly, touch up your styling so the items display correctly.